basicview.cpp

```diff
@@ -18,38 +18,77 @@ limitations under the License.
 
 #include <QVBoxLayout>
 
+#include <QFontDatabase>
+#include <QFont>
+#include <QDebug>
+#include <QChar>
+
+const QString startIcon = "\uf04b";
+const QString stopIcon = "\uf04d";
+const QString pauseIcon = "\uf04c";
+const QString deleteIcon = "\uf1f8";
+const QString reloadIcon = "\uf021";
+
+void BasicView::setFA(QWidget *wid)
+    {
+    if (QFontDatabase::addApplicationFont(":/images/FontAwesome.otf") < 0)
+        qWarning() << "FontAwesome cannot be loaded !";
+    QFont font;
```

Comment:
You can make this a file wide variable so you only have to set the family and pixel size once and then this func can only be `wid->setFont(fontAwesome)`